src/services/pcn-transparent-helloworld/datamodel/transparenthelloworld.yang

```diff
@@ -23,7 +23,7 @@ module transparenthelloworld {
       enum SLOWPATH;
     }
     default PASS;
-    description "Action performed on ingress packets";
+    description "Action performed on ingress packets (DROP/PASS/SLOWPATH; default: PASS)";
```

Comment:
I think information about possible values and default should not be part of the description.
Maybe the help generation could be improved to generate this.

Reply:
Right point, Mauricio. Moved now in the "example" section of the YANG. However, the example is not printed on screen by the help, so we should probably modify the command line to include that as well.